tests/testthat/helper-contexts.R

```diff
@@ -1,7 +1,7 @@
 with_fake_input <- function (input, expr) {
     with_mock(
         `crunch:::is.interactive`=function () return(TRUE),
-        `base::readline`=function (...) input,
+        `crunch:::read_input`=function (...) input,
```

Comment:
The `crunch:::` doesn't appear to be strictly necessary here. I tried removing them and running (against both release and master branch `testthat`) and the examples don't have them: https://github.com/hadley/testthat/blob/master/R/mock.R#L34

However, I don't mind leaving them in since they make it more explicit what's going on and make sure we aren't accidentally mocking from another namespace.

Reply:
_As it turns out_ `R CMD check` fails if you remove the `crunch:::` from these, like `Function is.interactive not found in environment testthat.`